commit.go

```diff
@@ -142,7 +142,7 @@ func commitsCount(repoPath, revision, relpath string) (int64, error) {
 	}
 
 	if isFallback {
-		return int64(len(strings.Split(stdout, "\n"))), nil
+		return int64(strings.Count(stdout, "\n")), nil
```

Comment:
Can you verify that the result is actually correct? I guess this will be 1 smaller than the actual.